docs/en/app-server/nginx+RR/Dockerfile

```diff
@@ -5,16 +5,12 @@ COPY --from=composer:latest /usr/bin/composer /usr/local/bin/composer
 COPY --from=roadrunner /usr/bin/rr /usr/local/bin/rr
 COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
 
-RUN mkdir /src
-COPY worker.php /src
-COPY .rr.yaml /src
-COPY composer.json /src
+RUN install-php-extensions sockets
+
+ENTRYPOINT ["rr"]
```

Comment:
Could you please put the `ENTRYPOINT` and the end of the file, `WORKDIR` at the beginning and use an absolute path `/src` (it is used to be more verbose, since this is a documetation).

Reply:
Done. Just rolled back that weird constructions